src/DevChatter.Bot/Startup/SetUpDatabase.cs

```diff
@@ -65,13 +65,61 @@ private static void EnsureInitialData(IRepository repository)
                 repository.Create(GetInitialQuizQuestions());
             }
 
+            CreateInitialRouletteSettingsIfNeeded(repository);
```

Comment:
If you move this below the missingCommandWords, you should be able to lookup the command word by type.